lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -205,6 +205,10 @@ def _init_axis(self):
         for ax in self.xaxis, self.yaxis, self.zaxis:
             ax.init3d()
 
+    def get_zaxis(self):
+        '''Return the :class:`ZAxis<~.axis3d.Axis>` instance.'''
+        return self.w_zaxis
```

Comment:
You can get self.zaxis instead (in fact w_{x,y,z}axis can probably be deprecated as aliases of {x,y,z}axis...).

Reply:
As of now it seems `zaxis` is the alias of `w_zaxis`, so I did rather take the original than the alias here. 
One could probably deprecate one of them, but there is for sure a lot of code out in the wild that uses either of them. I won't take the decision of which group to annoy with such deprecation here.

Reply:
Ok, so I rearanged the code, such that it is more obvious which ones should stay. I think I won't do the deprecation here, because I'm not convinced it is necessary. But if someone feels like it, they might easily do that independently of this PR.